src/Pos/Communication/Limits.hs

```diff
@@ -92,6 +96,19 @@ instance MessageLimitedPure (AbstractHash Blake2b_224 a) where
 instance MessageLimitedPure (AbstractHash Blake2b_256 a) where
     msgLenLimit = 32
 
+instance MessageLimitedPure w => MessageLimitedPure (ProxySecretKey w) where
+    msgLenLimit = coerce (msgLenLimit @w)
+                    + 2 * (coerce (msgLenLimit @PublicKey))
+                    + (coerce (msgLenLimit @(Signature w)))
+
+instance MessageLimitedPure w => MessageLimitedPure (ProxySignature w a) where
+    msgLenLimit = coerce (msgLenLimit @w)
+                    + (coerce (msgLenLimit @PublicKey))
+                    + 2 * (coerce (msgLenLimit @(Signature w)))
+
+instance MessageLimitedPure EpochIndex where
+    msgLenLimit = 8
```

Comment:
That `Word64` is serialized to 8 bytes? Well, needs to be checked, but I believe we don't use any `VarInt` there and thus I'm pretty sure